source/includes/_build.md

```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
 
 ### How to run development environment
 
-``` shell
-/path/to/iroha/scripts/run-iroha-dev.sh
-```
+Run the script `run-iroha-dev.sh`, contained in the folder `scripts`: `sh .../iroha/scripts/run-iroha-dev.sh`
 
-You will be attached to interactive environment for development and testing with `iroha` folder mounted from host machine.
+After you execute this script, following things happen:
 
-Docker environment will be removed when you logout from the container.
+ 1. It checks if you don't have containers with Iroha already running. If yes — it will reattach you to interactive shell. Assuming if you don't have any, then:
+ 2. The script will download iroha-docker-develop, redis and postgres images. Iroha image contains all development dependencies, and is based on top of ubuntu:16.04.
+ 3. Three containers are created and launched.
+ 4. The user is attached to the interactive environment for development and testing with `iroha` folder mounted from the host machine.
 
-### Build `iroha` and run tests
+<aside class="notice">
```

Comment:
Does this line have any effect on resulting text?

Reply:
Yes, https://github.com/lord/slate/wiki/Markdown-Syntax

Reply:
<img width="778" alt="screen shot 2017-12-16 at 16 22 29" src="https://user-images.githubusercontent.com/11841667/34070822-801acaba-e27d-11e7-93c1-e8905f3ac29c.png">